atd-vze/src/views/Crashes/CrashDiagram.js

```diff
@@ -0,0 +1,186 @@
+import React, { useState } from "react";
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Button,
+  ButtonGroup,
+  Col,
+  Row,
+} from "reactstrap";
+import axios from "axios";
+import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
+import CrashDiagramModal from "./CrashDiagramModal";
+
+const CrashDiagram = props => {
+  const [rotation, setRotation] = useState(0);
+
+  // Need to revisit setting the s3 folder based on environment, manually setting to "staging" for PR testing
+  // const s3Folder = process.env.NODE_ENV === "production" ? "production" : "staging";
+  const s3Folder = "staging";
+
+  console.log(process.env.NODE_ENV);
+
+  const requestCR3 = () => {
+    const requestUrl = `${process.env.REACT_APP_CR3_API_DOMAIN}/cr3/download/${props.crashId}`;
+    const token = window.localStorage.getItem("id_token");
+
+    axios
+      .request(requestUrl, {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+      })
+      .then(res => {
+        const win = window.open(res.data.message, "_blank");
+        win.focus();
+      });
+  };
+
+  const rotate = event => {
+    setRotation(event.target.value);
+  };
+
+  const resetRotate = () => {
+    setRotation(0);
+  };
+
+  return (
+    <Card className="h-100">
+      <CardHeader>
+        <Row className="d-flex align-items-center">
+          <Col>Crash Diagram</Col>
+          <Col>
+            {props.isTempRecord ? (
+              <CrashDiagramModal
+                buttonTitle={[
+                  "CR-3 PDF Unavailable",
```

Comment:
This is cool - so the array is getting joined in CrashDiagramModal? Just curious about this magic 🪄

Reply:
Yeah, this was something I discovered last week but forgot to mention it in a dev sync! I'll mention it in the next one!

Reply:
✨